src/evaluation/evaluate-score.service.ts

```diff
@@ -1,18 +1,25 @@
 import { Injectable, NotFoundException } from '@nestjs/common';
-import { Answer, SubmittedTest } from './interfaces/submitted-test.interface';
+import {
+  SubmittedAnswer,
+  SubmittedTest,
+} from './interfaces/submitted-test.interface';
 import { DBClientFactory } from 'src/db-client/db-client.factory';
 import { Test } from 'src/test/dto/test.dto';
 import { Question } from 'src/test/dto/question.dto';
 import { EvaluationProducerService } from './producer/evaluation-producer.service';
 import {
   EvaluatedAnswer,
-  Evaluation,
+  EvaluatedTestStatus,
 } from './interfaces/evaluated-test.interface';
 
-enum Status {
+enum TestStatus {
```

Comment:
Can create enum file seperated and keep all these enums in that file like enums -> test.enum.ts

Reply:
created `eums` folder and moved the code inside the suggested file `test.enum.ts`